src/components/ImageVote.vue

```diff
@@ -113,6 +115,26 @@ export default {
       }
       return false
     },
+    promptBadImage (error) {
+      console.warn(error.message)
+      let self = this
+      this.$vs.dialog({
+        type: 'confirm',
+        color: 'danger',
+        title: `Verbindungsfehler`,
+        text: 'Zurzeit kann keine Verbindung hergestellt werden. Überprüfen Sie bitte das Netzwerk und versuchen Sie es später erneut.',
+        acceptText: 'Bestätigen',
+        cancelText: 'Abbrechen',
+        accept: function () {
+          self.voteCount--
```

Comment:
This should not be done at the point the dialog closes, otherwise we might get negative vote counts.